celesta-maven-plugin/src/main/java/ru/curs/celesta/plugin/maven/CursorGenerator.java

```diff
@@ -190,10 +228,22 @@ private static String calcClassName(GrainElement ge, String sourceFileNamePrefix
         }
     }
 
+    private static String getCurrentDate() {
+        return new SimpleDateFormat("yyyy-MM-dd HH:mm").format(new Date());
```

Comment:
`Date` and `DateFormat` classes are outdated since Java 8. Please use modern DateTime API instead, e.g.

```java
DateTimeFormatter.ISO_LOCAL_DATE_TIME.format(LocalDateTime.now());
```

Reply:
Addressed!